src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/x4decoder/common/decoder/monitoringmetric/AudioPairMetric.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2022 AVI-SPL, Inc. All Rights Reserved.
+ */
+package com.avispl.symphony.dal.avdevices.encoderdecoder.haivision.x4decoder.common.decoder.monitoringmetric;
+
+/**
+ * Set of audio pair metrics keys
+ *
+ * @author Harry
+ * @since 1.0
+ */
+public enum AudioPairMetric {
+
+	// Static metric
+	AUDIO_PAIR_MODE("Pair Mode"),
+	COMPRESSION("Compression"),
+	BITRATE("Bitrate"),
+	AV_SYNC_MS("Av Sync Ms"),
+	DISCONTINUITIES("Discontinuities"),
+	DECODE_ERRORS("Decode Errors"),
+	OUTPUT_ERRORS("Output Errors"),
+	SAMPE_RATE_IN("Sampe Rate In"),
```

Comment:
Sample instead of Sampe, right ?

Reply:
At the first time, I thought it was "sample" but the response from API is "sampe"